torch/csrc/autograd/variable.h

```diff
@@ -22,15 +22,18 @@ namespace torch {
 namespace autograd {
 struct Function;
 } // namespace autograd
-namespace jit { namespace tracer {
+namespace jit {
+namespace tracer {
 // Has to be forward declared because tracer_state.h has a dependency on
 // variable.h.
 struct ValueTracingStateElem;
 using ValueTracingState = std::list<ValueTracingStateElem>;
-}} // namespace jit::tracer
+} // namespace tracer
+} // namespace jit
```

Comment:
Come on man. Those changes are really unnecessary, and they're not even consistent with what we usually do. Why are they part of this diff?!

Reply:
The `.clang-format` got changed recently to un-compact namespaces. Since I rewrote this file I had it clang-formatted entirely at the time, and assumed it safe to re-format it. But I agree this goes against autograd style. I will fix it